Sources/OpenCombine/Publishers/OperatorSubscription.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  OperatorSubscription.swift
+//  
+//
+//  Created by Sergej Jaskiewicz on 26.06.2019.
+//
+
+internal class OperatorSubscription<Downstream: Subscriber>: CustomReflectable {
+    internal var downstream: Downstream
+    internal var upstreamSubscription: Subscription?
+
```

Comment:
I'm looking forward to tweaking my PRs to use this instead of what I've currently done in my other open ones.  I'm wondering if this should conform to `Subscription` also.  It seems like every subclass is saving the upstream subscription and then sending to the downstream.  We could the override that in Operators like `DropWhile` to request 1 instead of a different number.  Do you think there is enough meaningful overlap in the operators to do that?

Reply:
>Do you think there is enough meaningful overlap in the operators to do that?

I don't think so. This makes sense only for map-like operators, we don't have many of them.

I intentionally haven't provided any implementation of the `Subscription` protocol here to encourage implementors (including myself in the future) to really think about the semantics of the `request(_:)` method for every particular operator.

Reply:
Agreed with @broadwaylamb.